trusted-committer/04-uplevelling-community-members.asciidoc

```diff
@@ -1,41 +1,39 @@
+As a Trusted Committer, you are responsible for moving individuals along this continuum
+and upleveling their ability to make contributions. In this sense, Trusted Committers
+act as force multipliers in their community.
 
-As indicated earlier, it is helpful for Trusted Committers to engage in marketing their
-product and their community in order to increase the number of newcomers and
-consumers. They should communicate opportunities for making contributions to
-consumers and try to elicit and align the interests of potential https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_]
-with those of the community. What often works well is if Contributors were able
-to work on something that made their day job easier, e.g. development tools and automation.
+It is important for Trusted Committers to market their
+product and communities to increase the number of
+newcomers and consumers. They should also communicate opportunities to
+make contributions to consumers and try to elicit and align the
+interests of potential https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] with that of the community. What
+often works well is if https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] are able to work on something that
+benefits their department or role in the company. Development tools and automation are good examples.
 
-Finally, it is the Trusted Committer's responsibility to identify https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] with the
-potential to grow.  They should foster and nurture that growth by getting
-Contributors excited for tackling challenging tasks and to mentor or coach them
-while they are performing them. This is, in our opinion, the noblest
-responsibility a Trusted Committer has. It is rewarding for both the Contributor and the
-Trusted Committer alike. We have heard from Trusted Committers that mentoring and seeing people level up
-their abilities more than makes up for the fact that they have less time to
-actually spend writing software.
+Finally, it is the Trusted Committer's responsibility to identify and support https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] with the potential to grow 
+by encouraging them to tackle challenging tasks and helping them complete them. This is, in our opinion, a Trusted Committer's
```

Comment:
`help them` vs. `mentor or coach them while they are performing them`.
I like the focus on mentoring and coaching them here.

I added a commit suggestion of an alternative phrasing.
```suggestion
by encouraging them to tackle challenging tasks and mentor them towards completion. This is, in our opinion, a Trusted Committer's
```